content/guides/core-concepts/test-runner.md

```diff
@@ -167,6 +210,68 @@ _Note: Internally, the AUT renders within an iframe. This can sometimes cause
 unexpected behaviors
```

Comment:
This note can be a small info alert box since it's sharing some additional info and redirecting to another resource. I know we haven't standardized alert box usage yet, but it does make sense here. The same would apply to other places on this page where an italicized note is used.

Reply:
I put both italic notes into blue alert boxes, but now the second screenshot doesn't really have any content above it anymore, which feels weird to me, eg.

![image](https://user-images.githubusercontent.com/54051/144302653-90022172-679c-4aab-96f7-0d8a01d86207.png)

Reply:
Ok, change it back, but please make an issue in our backlog to discuss and set the standards for alert box usage (you may have already done this). We'll solve this once and for all.

Reply:
https://linear.app/cypress/issue/DX-760/set-standards-for-alert-box-usage